app/presenters/applicant_account_presenter.rb

```diff
@@ -0,0 +1,21 @@
+class ApplicantAccountPresenter
+  def initialize(applicant_bank = nil)
+    @applicant_bank = applicant_bank
+  end
+
+  def main_account_holder_name
+    @applicant_bank.bank_account_holders.last.full_name
+  end
+
+  def main_account_holder_address
+    @applicant_bank.bank_account_holders.last.addresses.first.values.join(', ')
+  end
+
+  def bank_accounts
+    @applicant_bank.bank_accounts
+  end
+
+  def name
+    @applicant_bank.name
+  end
```

Comment:
When you are just calling a method on a sub-object, a useful pattern is to use `attr_reader` to create a getter for the object, and then `delegate` to access the individual methods. Especially in a presenter as that also gives you the [options `:prefix` and `allow_nil`](https://api.rubyonrails.org/classes/Module.html#method-i-delegate). So:
```ruby
  attr_reader :applicant_bank
  delegate :name, :bank_accounts, to: :applicant_bank
```